package.json

```diff
@@ -9,8 +9,7 @@
         "start:dev": "npm run wds -- --config ./webpack.config.dev.js --mode development",
         "start": "simplehttpserver dist",
         "test": "jest",
-        "test:coverage": "npm run test -- --coverage",
-        "cypress:open": "cypress open"
+        "test:e2e": "cypress open"

```

Comment:
In this case, I meant to start your dev server + open cypress to run it, not only to open cypress